src/bbs2gh/Handlers/MigrateRepoCommandHandler.cs

```diff
@@ -495,10 +513,15 @@ private void ValidateUploadOptions(MigrateRepoCommandArgs args)
             {
                 throw new OctoshiftCliException("Either --aws-secret-key or AWS_SECRET_KEY environment variable must be set.");
             }
+
+            if (!GetAwsRegion(args).HasValue())
+            {
+                _log.LogWarning("Please consider providing --aws-region or AWS_REGION environment variable. It will be required in future releases.");
+            }
         }
-        else if (args.AwsAccessKey.HasValue() || args.AwsSecretKey.HasValue())
+        else if (new[] { args.AwsAccessKey, args.AwsSecretKey, args.AwsSessionToken, args.AwsRegion }.Any(x => x.HasValue()))
         {
-            throw new OctoshiftCliException("--aws-access-key and --aws-secret-key can only be provided with --aws-bucket-name.");
+            throw new OctoshiftCliException("Amazon S3 upload options can only be provided with --aws-bucket-name.");
```

Comment:
```suggestion
            throw new OctoshiftCliException("The AWS S3 bucket name must be provided with --aws-bucket-name if other AWS S3 upload options are set.");
```

Reply:
Addressed in 98de164b93e6c5ad0a9f24c52673cd80b4f9a7fc